src/main/user-input-history-manager.test.ts

```diff
@@ -15,6 +15,26 @@ describe(TestableUserInputHistoryManager.name, (): void => {
             expect(actual[0]).toBe(userInput);
             expect(manager.getIndex()).toBe(1);
         });
+
+        it("should only add an item if it's not the same as the previous", (): void => {
+
+            const manager = new TestableUserInputHistoryManager;
```

Comment:
I think there are parantheses missing. It should be `const manager = new TestableUserInputHistoryManager();`

Reply:
Yeah, that was my bad.